src/components/BorrowerProfile/LendCta.vue

```diff
@@ -222,20 +227,32 @@
 						All shares reserved
 					</p>
 					<hr
-						class="tw-hidden md:tw-block tw-border-tertiary tw-w-full tw-my-2"
+						class="md:tw-block tw-border-tertiary tw-w-full tw-my-2"
+						:class="[
+							{
+								'tw-hidden': isSticky,
+								'tw-block': !isSticky,
+							}
+						]"
 					>
 					<div
-						class="tw-flex lg:tw-justify-center tw-w-full"
-						:class="isLoggedIn ? 'tw-justify-between' : 'tw-justify-end'"
+						class="md:tw-block tw-flex lg:tw-justify-center tw-w-full"
+						:class="[
+							{
+								'tw-justify-between': isLoggedIn,
+								'tw-justify-end': !isLoggedIn,
+								'tw-hidden': isSticky,
+							}
+						]"
 					>
 						<loan-bookmark
 							v-if="isLoggedIn"
 							data-testid="bp-lend-cta-loan-bookmark"
 							:loan-id="loanId"
-							class="tw-hidden md:tw-inline-block lg:tw-hidden"
+							class="tw-inline-block md:tw-hidden"
```

Comment:
Previously this could only be hidden here b/c it would be shown in the other component. But now it's removed in the other component. We need to make sure it is visible for a logged in user on all screen sizes.

Reply:
The code here looks like it's going to be hidden on all screens except mobile...